cmd/troubleshoot/cli/root.go

```diff
@@ -42,10 +42,10 @@ from a server that can be used to assist when troubleshooting a server.`,
 
 	cmd.Flags().StringSlice("redactors", []string{}, "names of the additional redactors to use")
 	cmd.Flags().Bool("redact", true, "enable/disable default redactions")
-	cmd.Flags().Bool("collect-without-permissions", false, "always run troubleshoot collectors even if some require permissions that troubleshoot does not have")
+	cmd.Flags().Bool("collect-without-permissions", false, "always generate a support bundle, even if it some require additional permissions")
 
 	// hidden in favor of the `insecure-skip-tls-verify` flag
-	cmd.Flags().Bool("allow-insecure-connections", false, "don't verify TLS certs when retrieving spec and reporting results")
+	cmd.Flags().Bool("allow-insecure-connections", false, "when set, do not verify TLS certs when retrieving spec and reporting results")
```

Comment:
@CodeLingoBot capture avoid contractions in strings
In error or user-facing strings, avoid using contractions

Bad code:
msg := "don't do that"

Good code:
msg := "do not do that"